guides/source/active_record_callbacks.md

```diff
@@ -3,22 +3,31 @@
 Active Record Callbacks
 =======================
 
-This guide teaches you how to hook into the life cycle of your Active Record objects.
+This guide teaches you how to hook into the life cycle of your Active Record
+objects.
 
 After reading this guide, you will know:
 
 * When certain events occur during the life of an Active Record object
-* How to create callback methods that respond to events in the object life cycle.
-* How to create special classes that encapsulate common behavior for your callbacks.
+* How to create callback methods that respond to events in the object life
+  cycle.
+* How to create special classes that encapsulate common behavior for your
+  callbacks.
 
 --------------------------------------------------------------------------------
 
 The Object Life Cycle
 ---------------------
 
-During the normal operation of a Rails application, objects may be created, updated, and destroyed. Active Record provides hooks into this *object life cycle* so that you can control your application and its data.
+During the normal operation of a Rails application, objects may be created,
+updated, and destroyed. Active Record provides hooks into this *object life
+cycle* so that you can control your application and its data.
 
-Callbacks allow you to trigger logic before or after an alteration of an object's state.
+Callbacks allow you to trigger logic before or after an alteration of an
```

Comment:
These sentences feel almost like saying the same thing, so they can probably be condensed into one sentence. Such as: Callbacks are methods that get called at certain moments of an
object's life cycle allowing you to trigger logic before or after an alteration of an
object's state.